File: platforms/swift/examples/core/texture_mip_chain/level_count.swift
import FragmentColor
import Foundation

let pixels = Data(repeating: 0, count: 8 * 8 * 4)
let chain = try TextureMipChain.prepare(
    bytes: pixels,
    format: .rgba8UnormSrgb,
    size: Size(width: 8, height: 8, depth: nil)
)
let count = chain.levelCount()
let _ = count

File: platforms/web/examples/core/texture_mip_chain/level_count.js
import { TextureFormat, TextureMipChain } from "fragmentcolor";

const pixels = new Uint8Array(8 * 8 * 4);
const chain = TextureMipChain.prepare(pixels, TextureFormat.Rgba8UnormSrgb, [8, 8]);
const count = chain.levelCount();
const _ = count;